app/controllers/root_controller.rb

```diff
@@ -273,10 +259,42 @@ def treat_as_standard_html_request?
     !request.format.json? and !request.format.print? and !request.format.video?
   end
 
-  def local_transaction_details(artefact, authority_slug, snac)
-    raise RecordNotFound if snac.blank? && authority_slug.present?
+  def local_transaction_details(artefact, authority_slug)
```

Comment:
We can definitely try this when we get the LocalLinksManager API working. I'm a bit unclear how much the code would have to change. Currently we just produce a hash of local interaction data which we ultimately assign to `@interaction_details` [here](https://github.com/alphagov/frontend/pull/977/files#diff-132a19fd5e7a1edc0d9127204dfa48eaR138) and then access in various views. So unsure of the impact of merging this kind of data back into the artefact and accessing the relevant data from it instead of `@interaction_details`.